src/_main_/settings.py

```diff
@@ -27,8 +27,8 @@
 # ********  LOAD CONFIG DATA ***********#
 # DJANGO_ENV can be passed in through the makefile, with "make start env=local"
 DJANGO_ENV = os.environ.get("DJANGO_ENV","remote")
-RUN_SERVER_LOCALLY = False
-RUN_CELERY_LOCALLY = False
+RUN_SERVER_LOCALLY = True
```

Comment:
Rather than change settings.py - etter if you change the Makefile to set proper run config options